plugins.d/pip.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+echo "🐍 Python pip"
+pip freeze --local | grep -v '^\-e' | cut -d = -f 1  | xargs pip install -U
```

Comment:
I think this won't work if you happen to run `kymsu` inside a `virtualenv`. Maybe add some check to see if a virtualenv is active? Something like `if [[ -z $VIRTUAL_ENV && -z $PYENV_VERSION ]]; then`

Reply:
Might be better to call `pip install -U --upgrade-strategy eager` so dependencies also get updated.